### PR TITLE
fix: various one-shot startup optimizations 

### LINKED ISSUE #3804: Reduce helm probe settings for Mirror node components
### Feature to Enhance

Reduce the values for readinessProbe, livenessProbe and startupProbe for the Mirror node helm charts

### Requested Enhancement

.

### Business Need

.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
## Description

This PR aims to reduce the warm and cold startup time of `solo one-shot single deploy` by changing the default Helm configurations.

Optimizes `solo consensus node setup` warm startup time by caching the build zip locally in the solo cache directory and uploading it to the container.

Reduces warm startup times of:
- Mirror node from ~70s to ~30s
- Relay from ~30s to ~20s
- Explorer from ~15s to ~5s
- Block node: the startup time is highly variable, so I cannot say with certainty what the effect of the changes in the PR is.
Removes an unnecessary 10-second sleep time after the command has completed, but before finalization.
- Consensus node setup from ~30s to 4s

The warm startup time on main is at 216 seconds. After the optimizations, it is now 121 seconds.

### Related Issues

* Closes #3804
* Related to #

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

**Review by JeffreyDallas — CHANGES_REQUESTED** [review]

File: resources/mirror-node-values.yaml [hunk 1]
```diff
@@ -33,8 +33,16 @@ importer:
       cpu: 1000m
       memory: 2000Mi
   readinessProbe:
+    initialDelaySeconds: 0
```
Comment (jan-milenkov) [inline resources/mirror-node-values.yaml]: Same for those, as I wrote in the other comments, some small delay might be beneficial for performance

File: resources/relay-values.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+readinessProbe:
```
Comment (jan-milenkov) [inline resources/relay-values.yaml]: Same for those, as I wrote in the other comments, some small delay might be beneficial for performance

**Review by jan-milenkov — APPROVED** [review]

**Review by JeffreyDallas — APPROVED** [review]
